components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/resources/web/taxonomy/js/ns-editor.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+var DEFAULT_DIV_ID = 'nsEditor';
+
+//private method 
+function show_ns_editor(dID, id, mode, linkID) {
+
+    var sufix = "";
+    if (linkID != undefined && linkID != null && linkID != "null" && linkID != "") {
+        document.getElementById(linkID).style.display = "none";
+        sufix = "&linkID=" + linkID;
+    }
+
+    if (mode != undefined && mode != null) {
+        sufix = "&editorMode=" + mode;
+    }
+
+    var url = 'ns_editor-ajaxprocessor.jsp?currentID=' + id + '&divID=' + dID + sufix;
+
+    var loadingContent = "<div id='workArea' style='overflow-x:hidden;'><div id='popupContent'><div class='ajax-loading-message'> <img src='../resources/images/ajax-loader.gif' align='top'/> <span>" + org_wso2_carbon_governance_lcm_ui_jsi18n["ns.editor.waiting.text"] + "</span> </div></div></div>";
+    CARBON.showPopupDialog(loadingContent, org_wso2_carbon_governance_lcm_ui_jsi18n["ns.editor.title"], 300, false, null, 550);
+
+    jQuery("#popupContent").load(url, null,
+        function (res, status, t) {
+            if (status != "success") {
+                CARBON.showWarningDialog(org_wso2_carbon_governance_lcm_ui_jsi18n["ns.editor.load.error"]);
+            }
+        });
+
+
+    return false;
+}
+
+function showNameSpaceEditorOnDiv(dID, id) {
+    return show_ns_editor(dID, id);
+}

```

Comment:
This is an external library,right?

We should not make any changes to this file.